tests/src/JIT/HardwareIntrinsics/X86/Sse2/ConvertScalarToVector128Int64.cs

```diff
@@ -22,16 +22,16 @@ static unsafe int Main(string[] args)
 
             if (Sse2.IsSupported)
             {
-                using (TestTable<long> longTable = new TestTable<long>(new long[2] { 1, -5 }, new long[2]))
+                using (TestTable<long> longTable = new TestTable<long>(new long[2] { 1, -758765098768743212 }, new long[2]))
```

Comment:
Unrelated test changes?

Reply:
Yeah, it slipped from other PR work while changing branches. Will remove.